### DISCUSSION TITLE
I am using ddt4all version 3.0.1. I could not find Clio 5.

### DISCUSSION THREAD

**Discussion by Deswa16** [cedricp/ddt4all#1087] [Q&A] [answered]
I am using ddt4all version 3.0.1. I could not find Clio 5. It does not say BJA ​​or XJA in the vehicle list. What was the name of Clio 5 changed to?

**Comment by matzel82** [comment]
You can proceed a Scan ? So you can scan your car automatically…or not ?

**Comment by Deswa16** [comment]
I didn't have a chance to try the new version. I looked at the changes in
demo mode. Sandero logan 2 part is gone. Clio V part Bja Xja part is gone.
I couldn't find it. Can you write the new names if you know them?

27 Nis 2024 Cmt 21:05 tarihinde matzel82 ***@***.***> şunu
yazdı:

**Comment by Deswa16** [comment]
I didn't have a chance to try the new version. I looked at the changes in demo mode. Sandero logan 2 part is gone. Clio V part Bja Xja part is gone. I couldn't find it. Can you write the new names if you know them?

**Comment by KarelSvo** [comment]
> I didn't have a chance to try the new version. I looked at the changes in demo mode. Sandero logan 2 part is gone. Clio V part Bja Xja part is gone. I couldn't find it. Can you write the new names if you know them?

@Furtif   has removed the understandable vehicle type list from ddt4all and replaced it with the error-filled list from the ddt2000 database.

Alternatively, you can use [PyRen](https://transfiles.ru/0yus7) PyRen is a combination of CLIP and ddt2000

**Reply by Deswa16** [reply]
Is there any way to solve this problem

**Reply by matzel82** [reply]
When he use the ecu.zip database ???

**Reply by Deswa16** [reply]
I moved the ecu.zip and ddt2000 folder in the old version folder to the folder of the ddt4all program I newly installed, and this is what happened next

**Reply by KarelSvo** [reply]
https://github.com/cedricp/ddt4all/blob/winXP/ddt4all.py#L38

This was the original vehicle list.
They would have only had to be added manually instead of blindly copying the ddt2000 list with all its errors.

**Reply by Furtif** [reply, marked as answer]
xJA _ Clio (C1A) car name in combobox is "Renault Clio (C1A)"

**Comment by KarelSvo** [comment]
User-friendly software should be self-explanatory, at least in terms of its basic functions.
The scan results for this type of vehicle are interesting. It shows the next ddt4all 3.0.1 problem.

**Comment by KarelSvo** [comment]
>  Sandero logan 2 part is gone. 

As with many vehicle types, the wrong name from the ddt2000 database is displayed. Furtif now must to do vehicle translater :-D 
My old Python 2 version was more understandable. https://www.drive2.ru/b/498093336985338243/ 
 It has made ddt4all very popular thanks to its convenient handling. https://www.google.com/search?q=https%3A%2F%2Fwww.drive2.ru%2Fb%2F498093336985338243%2F
Especially the Dacia version with a summary of the most popular changes.

**Reply by AndyIreland** [reply]
I like Python 2 DDT4ALL much better it just does what its meant to do and I find it more stable

**Reply by Furtif** [reply]
@KarelSvo @AndyIreland 
it all depends on the content of projects.json or how it is generated or edited:

```
"Renault Clio (C1A)": {
   "code": "XJA",
   "addressing": {
    "00": [
     "CAN",
     "Primary CAN network"
 ````
 You can edit it like this:
```
 "Renault Clio V - [XJA]": {
   "code": "XJA",
   "addressing": {
    "00": [
     "CAN",
     "Primary CAN network"
 ````
 Each project has the code that corresponds
```
   "code": "XJA",
```
https://raw.githubusercontent.com/cedricp/ddt4all/master/ddt4all_data/projects.json
 
 as projects.json is self generated I just use the brand (renault, nissan, dacia) and the name that is in db the name used in dtt2000, the code or [xja] etc address, dnat, snat of each car, the versions old ones just use a fixed list be it cars dnat or snat or even addresses when using projects.json each car contains the real values that each car has in dtt2000, I don't see where the problem is!

**Reply by Furtif** [reply]
https://github.com/cedricp/ddt4all/blob/master/parameters.py#L1950 you can give the code value in project name and generate projects.json this way

**Reply by Furtif** [reply]
More or less this:
```
x06                => vehicles\x06\addressing.xml              => OK
x07                => vehicles\x07\addressing.xml              => OK
x07Ph2             => vehicles\x07Ph2\addressing.xml           => OK
x09                => vehicles\x09\addressing.xml              => OK
x1316              => vehicles\x1316\addressing.xml            => OK
x1317              => vehicles\x1317\addressing.xml            => OK
x10                => vehicles\x10\addressing.xml              => OK
xCB                => vehicles\xCB\addressing.xml              => OK
xCBPh2             => vehicles\xCBPh2\addressing.xml           => OK
x10Ph2             => vehicles\x10Ph2\addressing.xml           => OK
x33                => vehicles\x33\addressing.xml              => OK
x44                => vehicles\x44\addressing.xml              => OK
x44Ph2             => vehicles\x44Ph2\addressing.xml           => OK
xBA                => vehicles\xBA\addressing.xml              => OK
xBG                => vehicles\xBG\addressing.xml              => OK
xBGPh2             => vehicles\xBGPh2\addressing.xml           => OK
xBB                => vehicles\xBB\addressing.xml              => OK
x35                => vehicles\x35\addressing.xml              => OK
x65                => vehicles\x65\addressing.xml              => OK
xBC                => vehicles\xBC\addressing.xml              => OK
x77                => vehicles\x77\addressing.xml              => OK
x77Ph2             => vehicles\x77Ph2\addressing.xml           => OK
x87                => vehicles\x87\addressing.xml              => OK
x87Ph2             => vehicles\x87Ph2\addressing.xml           => OK
x85                => vehicles\x85\addressing.xml              => OK
x98                => vehicles\x98\addressing.xml              => OK
x98Ph2             => vehicles\x98Ph2\addressing.xml           => OK
xJA                => vehicles\xJA\addressing.xml              => OK
xJAPh2             => vehicles\xJAPh2\addressing.xml           => OK
xFI                => vehicles\xFI\addressing.xml              => OK
x64                => vehicles\GenericAddressing.xml           => OK
xJF                => vehicles\xJF\addressing.xml              => OK
xCC                => vehicles\xCC\addressing.xml              => OK
xCD                => vehicles\xCD\addressing.xml              => OK
x84                => vehicles\x84\addressing.xml              => OK
x84bugABS          => vehicles\x84\addressingWithoutABS.xml    => OK
x84ABSonly         => vehicles\x84\addressingABS.xml           => OK
x84Ph2             => vehicles\x84Ph2\addressing.xml           => OK
x95                => vehicles\x95\addressing.xml              => OK
x95Ph2             => vehicles\x95Ph2\addressing.xml           => OK
x38                => vehicles\x38\addressing.xml              => OK
xJL                => vehicles\xJL\addressing.xml              => OK
xJLPh2             => vehicles\xJLPh2\addressing.xml           => OK
xJH                => vehicles\xJH\addressing.xml              => OK
xFA                => vehicles\xFA\addressing.xml              => OK
xFAPh2             => vehicles\xFAPh2\addressing.xml           => OK
xFB                => vehicles\xFB\addressing.xml              => OK
xFBPh2             => vehicles\xFBPh2\addressing.xml           => OK
HFEPh2             => vehicles\HFEPh2\addressing.xml           => OK
xHN                => vehicles\xHN\addressing.xml              => OK
xHNPh2             => vehicles\xHNPh2\addressing.xml           => OK
xHC                => vehicles\xHC\addressing.xml              => OK
HFE                => vehicles\HFE\addressing.xml              => OK
xFF                => vehicles\xFF\addressing.xml              => OK
xFFPh2             => vehicles\xFFPh2\addressing.xml           => OK
xFJ                => vehicles\xFJ\addressing.xml              => OK
xEF                => vehicles\xEF\addressing.xml              => OK
xJB                => vehicles\xJB\addressing.xml              => OK
xJBPh2             => vehicles\xJBPh2\addressing.xml           => OK
xJP                => vehicles\xJP\addressing.xml              => OK
xJE                => vehicles\xJE\addressing.xml              => OK
xZH                => vehicles\xZH\addressing.xml              => OK
xZHPh2             => vehicles\xZHPh2\addressing.xml           => OK
x56                => vehicles\GenericAddressing.xml           => OK
x66                => vehicles\GenericAddressing.xml           => OK
xZG                => vehicles\xZG\addressing.xml              => OK
xZI                => vehicles\xZI\addressing.xml              => OK
xZGPh2             => vehicles\xZGPh2\addressing.xml           => OK
xZGPh3             => vehicles\xZGPh3\addressing.xml           => OK
xZJ                => vehicles\xZJ\addressing.xml              => OK
xZJPh2             => vehicles\xZJPh2\addressing.xml           => OK
x74                => vehicles\x74\addressing.xml              => OK
x74Ph2             => vehicles\x74Ph2\addressing.xml           => OK
x47                => vehicles\x47\addressing.xml              => OK
x91                => vehicles\x91\addressing.xml              => OK
x91Ph2             => vehicles\x91Ph2\addressing.xml           => OK
x91Ph3             => vehicles\x91Ph3\addressing.xml           => OK
x96                => vehicles\x96\addressing.xml              => OK
xFD                => vehicles\xFD\addressing.xml              => OK
xFDPh2             => vehicles\xFDPh2\addressing.xml           => OK
x43                => vehicles\x43\addressing.xml              => OK
x54                => vehicles\GenericAddressing.xml           => OK
x73                => vehicles\x73\addressing.xml              => OK
x73Ph2             => vehicles\x73Ph2\addressing.xml           => OK
x81                => vehicles\x81\addressing.xml              => OK
x81Ph2             => vehicles\x81Ph2\addressing.xml           => OK
x81Ph3             => vehicles\x81Ph3\addressing.xml           => OK
x81Ph4             => vehicles\x81Ph4\addressing.xml           => OK
x94                => vehicles\GenericAddressing.xml           => OK
xFC                => vehicles\xFC\addressing.xml              => OK
xFCPh2             => vehicles\xFCPh2\addressing.xml           => OK
xFG                => vehicles\xFC\addressing.xml              => OK
x70                => vehicles\x70\addressing.xml              => OK
x70Ph3             => vehicles\x70Ph3\addressing.xml           => OK
x76                => vehicles\x76\addressing.xml              => OK
xFK                => vehicles\xFK\addressing.xml              => OK
xJK                => vehicles\xJK\addressing.xml              => OK
x61                => vehicles\x61\addressing.xml              => OK
u60                => vehicles\u60\addressing.xml              => OK
x61Ph2             => vehicles\x61Ph2\addressing.xml           => OK
x82                => vehicles\x82\addressing.xml              => OK
x82Ph2             => vehicles\x82Ph2\addressing.xml           => OK
x83                => vehicles\x83\addressing.xml              => OK
x83Ph2             => vehicles\x83Ph2\addressing.xml           => OK
x83Ph3             => vehicles\x83Ph3\addressing.xml           => OK
x62                => vehicles\x62\addressing.xml              => OK
x62Ph2             => vehicles\x62Ph2\addressing.xml           => OK
xDC                => vehicles\xDC\addressing.xml              => OK
xDD                => vehicles\xDD\addressing.xml              => OK
xDE                => vehicles\xDE\addressing.xml              => OK
x24                => vehicles\x24\addressing.xml              => OK
x89                => vehicles\GenericAddressing.xml           => OK
x52                => vehicles\x52\addressing.xml              => OK
xJI                => vehicles\xJI\addressing.xml              => OK
x92                => vehicles\x92\addressing.xml              => OK
xHA                => vehicles\xHA\addressing.xml              => OK
x1312              => vehicles\GenericAddressing.xml           => OK
xHAPh2             => vehicles\xHAPh2\addressing.xml           => OK
xGA                => vehicles\xGA\addressing.xml              => OK
x79                => vehicles\x79\addressing.xml              => OK
x79Ph2             => vehicles\x79Ph2\addressing.xml           => OK
x1310              => vehicles\x1310\addressing.xml            => OK
xJC                => vehicles\xJC\addressing.xml              => OK
xJD                => vehicles\xJD\addressing.xml              => OK
x90                => vehicles\x90\addressing.xml              => OK
Nova               => vehicles\GenericAddressing.xml           => OK
x67                => vehicles\x67\addressing.xml              => OK
x1316A             => vehicles\x1316A\addressing.xml           => OK
AS1                => vehicles\AS1\addressing.xml              => OK
DZ110              => vehicles\DZ110\addressing.xml            => OK
xR210              => vehicles\xR210\addressing.xml            => OK
March              => vehicles\GenericAddressing.xml           => OK
Primera            => vehicles\GenericAddressing.xml           => OK
Almera             => vehicles\Almera\addressing.xml           => OK
PZ1A               => vehicles\PZ1A\addressing.xml             => OK
PZ1C               => vehicles\PZ1C\addressing.xml             => OK
x02E               => vehicles\x02E\addressing.xml             => OK
x21B               => vehicles\x21B\addressing.xml             => OK
L21B               => vehicles\L21B\addressing.xml             => OK
x13A               => vehicles\x13A\addressing.xml             => OK
PY1B               => vehicles\PY1B\addressing.xml             => OK
P13C               => vehicles\GenericAddressing.xml           => OK
p32                => vehicles\p32\addressing.xml              => OK
x60A               => vehicles\x60A\addressing.xml             => OK
xNN                => vehicles\xNN\addressing.xml              => OK
LZ2A               => vehicles\LZ2A\addressing.xml             => OK
p33A               => vehicles\p33A\addressing.xml             => OK
p33B               => vehicles\p33B\addressing.xml             => OK
P42Q               => vehicles\P42Q\addressing.xml             => OK
P42R               => vehicles\P42R\addressing.xml             => OK
J32V               => vehicles\J32V\addressing.xml             => OK
RF90               => vehicles\RF90\addressing.xml             => OK
xGF                => vehicles\xGF\addressing.xml              => OK
xJN                => vehicles\xJN\addressing.xml              => OK
xJO                => vehicles\xJO\addressing.xml              => OK
KJA                => vehicles\GenericAddressing.xml           => OK
H45                => vehicles\h45\addressing.xml              => OK
w176               => vehicles\w176\addressing.xml             => OK
x60B               => vehicles\x60B\addressing.xml             => OK
w205               => vehicles\w205\addressing.xml             => OK
VS10               => vehicles\VS10\addressing.xml             => OK
VS11               => vehicles\VS11\addressing.xml             => OK
Edison             => vehicles\Edison\addressing.xml           => OK
xPIV_C1A           => vehicles\xPIV_C1A\addressing.xml         => OK
xPIV_C1AHS         => vehicles\xPIV_C1AHS\addressing.xml       => OK
xCOP_Before_C1A    => vehicles\xCOP_Before_C1A\addressing.xml  => OK
xCOP_C1A           => vehicles\xCOP_C1A\addressing.xml         => OK
xCOP_C1AHS         => vehicles\xCOP_C1AHS\addressing.xml       => OK
```

### Generated:
[projects.json](https://github.com/cedricp/ddt4all/files/15142351/projects.json)
Replace your projects.json with this one and you will see that it is as you want.

![image](https://github.com/cedricp/ddt4all/assets/11375035/a8bbf96f-e7a3-4b9d-84d5-e7f298c1418e)

**Reply by Furtif** [reply]
As I said, it all depends on how the json is generated, I just changed the way the project was named:
```
project_name = "[%s] - %s %s" % (str(code).upper(), name, p_name)
#if project_name in  dict["projects"].keys():
#    project_name = "%s %s (%s)" % (name, p_name, str(code).upper())
```
https://github.com/cedricp/ddt4all/blob/master/parameters.py#L1950

**Comment by KarelSvo** [comment]
The ddt2000 database contains vehicle type designations that do not correspond to the sales designation.
Automatic generation just confuses the user. Not everyone knows the type code of their vehicle.
The manually maintained vehicle type list in ddt4ll.py was better, the number of lines remains clear and can be sorted sensibly. No COP entries or inactive vehicle types.
Automatically prepending manufacturer names produces even more discrepancies.

**Reply by Furtif** [reply]
You are not obliged to use them, you can remove them from the json, or edit the project names in another way, I just do the complete dump of the current database.

**Reply by Furtif** [reply]
https://github.com/cedricp/ddt4all/discussions/1087#discussioncomment-9253494

**Comment by KarelSvo** [comment]
Nobody is obliged. However, the normal user is dependent on the basic functions being understandable. He is not a programmer, he has no background knowledge of what the automatically generated vehicle types really mean.
I don't know if the current solution is in the spirit of @cedricp .
Certainly not for most users.

**Reply by Furtif** [reply]
I think you speak for yourself, when users don't know they ask and most of the time they use issue instead of discussion, as I already told you, everyone's tastes and colors are different. Reading what you say leads me to believe that the official dtt2000 program is completely wrong.

**Comment by AndyIreland** [comment]
in all fairness to @cedricp he has only bought out the program itself DDT4ALL , fantastic work, he has nothing to do with the DDT2000 Database that is a separate bit altogether ... so if there is something wrong about the database or vehicles then that is nothing to do with Cedric , he has always said from the beginning that the ECU database has to be downloaded to use his program (DDT4ALL)

**Reply by Furtif** [reply]
yep https://github.com/cedricp/ddt4all?tab=readme-ov-file#informations

**Reply by Furtif** [reply]
so that the projects.json file is base or generated with the database, it is not because I changed the code so that the project (or car) uses the correct database addresses, that is, addressing.xml and the dnat snat of the same instead of always use fixed values for all cars (GenericAddressing.xml) which is bad! Just edit the projects.json file to your liking, and/or edit the way it is generated and create a different one, as I did above.

https://github.com/cedricp/ddt4all/discussions/1087#discussioncomment-9252344

replace your projects.json with what was generated, the list will be different, if it's not good just remove the ones you don't want with notepad ++ or rename the names of each project (leaving the address, snat, dnat of each)

**Reply by Furtif** [reply]
If you don't like the list with constructor or manufacter, you can generate it like this:

```
project_name = "[%s] - %s" % (str(code).upper(), p_name)
#if project_name in  dict["projects"].keys():
#    project_name = "%s %s (%s)" % (name, p_name, str(code).
```

### Generated:
[projects.json](https://github.com/cedricp/ddt4all/files/15142813/projects.json)

![image](https://github.com/cedricp/ddt4all/assets/11375035/6e0c0973-367b-4fd7-850f-12c472692489)

If you don't want some cars in the list, delete the corresponding folder in vehicle/xxxx and regenerate the projects.json file.

For information purposes, the file is present with a global dump, this does not mean that it pleases everyone, it could be deleted and said **"Don't forget to add the database and generate the list"**

The problem in general with the list is having to use the real database and not ecu.zip, you cannot create or generate json with just the zip, which is why it is in global format

**Reply by Furtif** [reply]
Command line to generates file projects.json is:

```
python parameters.py --dumpprojects
```

**Comment by KarelSvo** [comment]
The NOVA in your photo is an example of a useless line.
There are 0 control device files available for this.
I am not aware of a DNAT, SNAT problem.
I manually add  the list in elm.py if necessary.
Just like the vehicle list in ddt4all.py.
The json list is too confusing for me to correct manually.
I will again pack a simple Python 2 complete installer with the latest ecu.zip, vehicle list and SNAT-DNAT.
Backwards compatible up to XP.

**Reply by Furtif** [reply]
Ok maybe but is in DataBase https://github.com/cedricp/ddt4all/discussions/845#discussioncomment-6144566 and in DTT2000 

![image](https://github.com/cedricp/ddt4all/assets/11375035/7f76ba23-72d0-42d7-80cc-228da207c50b)

Now I don't know which version you use as the database, a lot depends on that

**Comment by KarelSvo** [comment]
2022-10-25T18:27:52
You can see in the ddt2000 Builder that nothing is assigned there.
Also affects other entries in the projects list.

**Reply by Furtif** [reply]
Yes right, and I am not responsible for the content.

Just auto generates the projects.json if there is news.

![image](https://github.com/cedricp/ddt4all/assets/11375035/7cdf2055-b623-4b76-81cc-d376f7140e1e)

![image](https://github.com/cedricp/ddt4all/assets/11375035/1db07936-face-4f9b-92e2-ba0444bfaa4d)

There is no news in Ecu Download

**Reply by Deswa16** [reply]
It uses the same database as Mitsubishi Cold Clio 5. It is produced in Oyak Renault factory in Turkey. Can you make an addition about it, edit the list, download it and use it, or make a file such as Clio V and Clio V ph2 and submit it to us?

**Reply by Furtif** [reply]
@Deswa16 Clio 5 ph2 is Renault Clio (C1A) Phase2 in combobox, Then I'm not the one who edits the database

**Reply by Deswa16** [reply]
I understood from what you wrote that you edited it, but I just wanted to ask a question, do you have a chance to do this?

**Reply by Furtif** [reply]
@Deswa16 I didn't edit it, I just generated the projects.json file, but that depends on the content of the official DataBase

**Reply by Furtif** [reply]
https://github.com/cedricp/ddt4all/blob/master/ddt4all_data/projects.json 

Content the names of the cars in projects.xml and the addresses, dnat, snat respective to each project, can be generated in another way as I did above https://github.com/cedricp/ddt4all/discussions/1087#discussioncomment-9252344 If you prefer this model, replace the projects.json file generated above

**Reply by Furtif** [reply]
@Deswa16 You can edit to your liking, see https://github.com/cedricp/ddt4all/discussions/1087#discussioncomment-9253096 Or edit the projects.json file yourself, instead of clio V or c1A (real name in addressing.xml) you can put another

**Comment by KarelSvo** [comment]
> I like Python 2 DDT4ALL much better it just does what its meant to do and I find it more stable

[PY2](https://transfiles.ru/fbfak) to test

edited vehicle list
current ecus, snat, dnat
File editor removed and renamed to Demo
XP-11

**Reply by Furtif** [reply]
It's a bit regrettable, criticizing the developers, giving an "exe" instead of updating the 'famous xp branch' but you know that

**Reply by KarelSvo** [reply]
https://github.com/cedricp/ddt4all/discussions/1088#discussioncomment-9286930

update for easier use

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/pull/1091

**Comment by KarelSvo** [comment]
![img_525](https://github.com/cedricp/ddt4all/assets/30380693/e0e9d8d4-41b2-4ef3-808d-8afa6ad9448c)

![img_526](https://github.com/cedricp/ddt4all/assets/30380693/6d51ac7a-c7f7-4227-b6e9-0a802aad7277)

![img_527](https://github.com/cedricp/ddt4all/assets/30380693/6f59a5de-acf1-464a-a4ed-c66eebd5e0a7)


The normal user does not want to collect software parts, but simply install, start, see what to do next and simply find his vehicle model in the selection list. That's exactly what this file does. @cedricp ddt4all versions fulfilled the last point, the current versions no longer do. This criticism should be justified.
https://github.com/cedricp/ddt4all/commit/9abce59fc2ee067ebabd2f28c97ccd404dd671e8

**Reply by Furtif** [reply]
Yep lol but this is repo development XD

**Reply by Furtif** [reply]
If you don't like 3.0.1 there is more version in releases tab you can use another one....

**Reply by Furtif** [reply]
![Capture d’écran 2024-05-01 à 19 11 23](https://github.com/cedricp/ddt4all/assets/11375035/d9a967ea-df5d-40e9-9e12-83c6a3f5f350)
dark mode need still some litle small stuffs in macOs

**Reply by Furtif** [reply]
![Capture d’écran 2024-05-01 à 19 16 14](https://github.com/cedricp/ddt4all/assets/11375035/cbaa16ea-17dd-43ad-8ab0-fd1cc4b2dfe8)

**Reply by Furtif** [reply]
@KarelSvo 
![image](https://github.com/cedricp/ddt4all/assets/11375035/fd1b6585-8cd1-4f43-b5e7-e75e4c315ff2)
This is not a specific development for a single platform, you have your version (5.x very advanced it seems), it is derived from this github, there are tags in the [releases](https://github.com/cedricp/ddt4all/releases) if you don't like one use the other, you propose the yours (I'm not against it and I've seen many similar things without making a fuss). Do you think that me or @cedricp will go to other gits, or platforms, or other sites to announce other things, or whatever they do, it's not good, you can even comment on pyren, I didn't defame anything, instead I proposed a pr (participate [Pyren-add packs](https://gitlab.com/py_ren/pyren/-/commit/d9fa28ea3a51c4efc77683ef950affce83457ef0), instead of criticizing) to do something against it, or to advertise it? (besides your versions being all dependent and/or based on what was done)? Or criticize what is done? If you are stuck on the XP version that doesn't worry me, besides there is a branch, a tag and a version for that, if you are honest you should contribute to the code, if you like XP you would dedicate yourself to it, proposing something, Instead of always coming with 4 stones in hand, criticizing what is done, you would make this branch active and propose what they do and in the next releases have it installed as they love.

**Reply by Furtif** [reply]
I hope you don't mention me anymore, without it being truly necessary, neither me nor the creator of this repo.

**Reply by Furtif** [reply]
On the other hand, this is a development platform, not for users to have everything, you have your forums or websites or YouTube pages that have nothing to do with the github platform, or any contribution to development.

This platform is for the community to make contributions by taking the base and adding improvements as they think.

Take any release and use it, but if you think changes/improvements can be made please 'customise' your version and post for review.  In any case, there is no obligation for you to use the database as it is - you can make any changes to your local version as you want.

Github at its core is a developer platform - not a consumer outlet

**Comment by KarelSvo** [comment]
If development reduces user-friendliness, whoever did it will always face criticism. This is completely normal. The users know that it wasn't always like this  https://www.google.com/search?q=https%3A%2F%2Fwww.drive2.ru%2Fb%2F498093336985338243%2F

**Reply by Furtif** [reply]
And you continue with advertising that has nothing to do with this?

I completely don't give a shit about what Google gives, if your forum or website appears ok... ready, but stop being like that, come in the source code repos, and everything else to make it visible? ddt4all git also appears, and don't think that people are that stupid, when they see a version of 5.x and open and active source code in 3.x

**Reply by Furtif** [reply]
https://www.google.com/search?q=ddt4all

is github right ..

**Reply by Furtif** [reply]
When you come to your senses, come back

**Reply by M3G4THEKING** [reply]
Sorry but this type of response need to be stopped i gonna reply to a such "sh....t" response from @KarelSvo . 
"If development reduces user-friendliness, whoever did it will always face criticism. This is completely normal. The users know that it wasn't always like this blablabla.",  Have your read the README file? SAYS " Notes:
This application is work in progress, so be very careful when using expert mode. Using the application in non expert mode should not be harmful for your vehicle (leave the expert mode button released).

**Important :

Do not use this software if you don't have a strong knowledge of how a CAN network (or ECU) works, you can really do bad things with it, especially if you're working on a vehicle

The author declines all responsibility about a bad use of this tool. You are the only responsible**

This tool is mainly aimed for CAN ISO_TP network study" and more " Informations
DataBase not included, do not forget to install database as ecu.zip or full mode in to root clone repo.

Report bugs
Report bugs you found in [issues](https://github.com/cedricp/ddt4all/issues). In order to help us fix the problem, please take a screenshot of the error you get and also attach your log file (under the Logs folder) as well. Add [Bug] to the title to help us quickly identify the category of the issue.

Suggestions/ideas
Tell us what you think we can do better in [discussions](https://github.com/cedricp/ddt4all/discussions). Give detailed discription to help us understand what you are looking for. Add [Suggestion] to the title to help us quickly identify the category of the issue. Your suggestion might not be accept, but hey, maybe we will accept your suggestion next time! :)

Legal Disclaimer
This Website and Project is in no way affiliated with, authorized, maintained, sponsored or endorsed by ANYONE. This is an independent and unofficial project for educational use ONLY. Do not use for any other purpose than education, testing and research.

Happy CAN-Hacking :)

To make this application more reliable, I need to buy hardware, cables and devices, so please consider contributing by making a donation (hardware or money). Of course you can contribute by filling bug reports and sending patches.". If users dont like something, maybe they can creat some #Pr to DDT4ALL. Stop talking shit and do something , if you dont know how to start coding go learn, ask help and never CRISTICISM  if you dont do better. If you dont like creat your OWN DDT4ALL and BYE BYE.

**Reply by Furtif** [reply]
@M3G4THEKING he has already created, but he prefers to break the patience of those who are calm ... and come here to advertise..

**Comment by KarelSvo** [comment]
The title of this topic is self-evident.

**Reply by Furtif** [reply]
is... and belongs to this repo not your forums

**Reply by Furtif** [reply]
and answered, the person who asked the question validated it

**Reply by Furtif** [reply]
is marked answered before you come to propose something that has nothing to do with the repo

**Reply by Furtif** [reply]
you talk about titles (topics usual in forums), etc. and you don't even write in github, you know, you always have to open a new field instead and follow it.

**Comment by Deswa16** [comment]
Hello, I have the current vai database of DDT2000 4.0.9 version. How can I convert it to ecu.zip file?

**Reply by AndyIreland** [reply]
download something like winzip utility or winrar and zip it up and name it to ecu.zip

**Reply by Deswa16** [reply]
> winzip yardımcı programı veya winrar gibi bir şey indirip sıkıştırın ve adını ecu.zip yapın

I tried but the program doesn't see the brains that way.

**Reply by rlinkFORfree** [reply]
Open the ddt4all File menu and select the zip database option.

Did you purchase your ddt2000 version online? Then it's not a new database.

**Comment by Renotech1** [comment]
What do you mean you allready have clio6 xml ? Thanks

Op zo 21 sep 2025, 07:33 schreef Deswa16 ***@***.***>:

**Comment by Deswa16** [comment]
Thanx KarelSvo sorunu çözdü
